_posts/2017-12-23-opencv-3-4.html

```diff
@@ -0,0 +1,59 @@
+---
+date: 2017-12-23 14:00:00 +0300
+layout: post
+permalink: opencv-3-4.html
+published: true
+short: "OpenCV 3.4 has been released!"
+title: OpenCV 3.4
+---
+<img class="center" src="{{ site.baseurl }}{% link assets/news/opencv_ny.jpg %}" alt="OpenCV logo on the present box near the Cristmas tree"/>
+
+<p>We've released OpenCV 3.4.</p>
```

Comment:
<p>Right before the Christmas and New Year holidays, we are glad to present the latest and the greatest OpenCV 3.4.</p>